xray_vision/backend/mpl/cross_section_2d.py

```diff
@@ -278,6 +278,22 @@ class CrossSection(object):
     interpolation : str, optional
         Interpolation method to use. List of valid options can be found in
         CrossSection2DView.interpolation
+    vert_loc : str, optional
+        set to 'left' by default, can specify 'right' to move the verical
+        axis to the right side of the image
+    horiz_loc : str, optional
+        set to 'top' by default, can specify 'bottom' to move the horizontal
+        axis below the image
+    title : str, optional
+        title for the figure
+    vert_label : str, optional
+        label that describes the x axis of the vertical plot
+    horiz_label : str, optional
+        label that describes the x axis of the horizontal plot
+    extent_labels : list, optional
```

Comment:
- Maksim and I discussed the ```extent``` name, we just thought that if it were ```extent``` only there could be some confusion as the actual extent is not set - the image just appears as so. This is because when using actual ```extent``` the image would not fill the entire subplot. I do agree to change the type from a ```list``` to ```scalars``` with a more fitting description. 

- To go along with that we tried using the ```self.init_artists``` function to pass in ```extent``` but we encountered the error described above. Ideally ```extent``` would actually be set but we couldn't figure out how to do so. If this were figured out the ```extent_labels``` name could just simply be ```extent``` and it would make much more sense.

- I will be sure to check out moving the extent kwarg to the ```update_img``` function and look into aligning the plot lines - thanks for the feedback!